core/src/test/scala/org/apache/spark/sql/delta/DeltaWriteConfigsSuite.scala

```diff
@@ -535,13 +563,37 @@ class DeltaWriteConfigsSuite extends QueryTest
                   if (tblproperties_was_set) tblproperties_no_prefix.toString else "N/A",
                   if (tblproperties_was_set) tblproperties_prefix.toString else "N/A",
                   config.mkString(",")
-                ))
+                )
               }
             }
           }
         }
       }
     }
   }
-
 }
+
+// Need to be outside to be stable references for Spark to generate the case classes
+case class DeltaFrameStreamAPITestOutput(outputLocation: String,
```

Comment:
Could you move `outputLocation: String,` to the next line, such as
```
case class DeltaFrameStreamAPITestOutput(
    outputLocation: String,
    outputMode: String,
    containsNoPrefixOption: Boolean,
    containsPrefixOption: Boolean,
    config: String)
```
The current format makes it easy to miss the first parameter when reading the code.